docker-compose.yml

```diff
@@ -3,12 +3,10 @@ version: "3.7"
 services:
   breedbase:
     image: breedbase/breedbase:v0.17
-    restart: always
     depends_on:
       - breedbase_db
     container_name: breedbase_web
 
   breedbase_db:
-    image: breedbase/breedbase_pg12.4:v0.5
-    restart: always
+    image: breedbase/pg:11
```

Comment:
breedbase/pg:11 doesn't appear to exist. I tried breedbase/pg:10 (which is the same image as breedbase/pg:latest), and it at least started, though then the web_usr doesn't appear to have access to the breedbase database (even after granting LOGIN privilege, manually setting a password, and updating sgn_local.conf to reflect this password).